editor/index.js

```diff
@@ -25,6 +26,19 @@ export function createEditorInstance( id, post ) {
 		blocks: wp.blocks.parse( post.content.raw ),
 	} );
 
+	// Each property that is set in `post-content.js` (other than `content`
+	// because it is serialized when a save is requested) needs to be
+	// registered as an edit now.  Otherwise the initial values of these
+	// properties will not be properly saved with the post.
+	store.dispatch( {
+		type: 'EDIT_POST',
+		edits: {
+			title: post.title.raw,
+			...omit( post, 'title', 'content' ),
+		},
+		shouldMarkDirty: false,
+	} );
```

Comment:
I see `edits` as more like "the attributes of the post that need to be saved, but haven't yet".  `status: 'draft'` is a good one, along with whatever other defaults we choose for things like visibility.  I think this might keep our code a bit cleaner because we can use the same logic to read the current state of the post, for our controls UI.

I don't think we should send the entire `currentPost` output with each save request.  I'm aware of at one core bug that we would hit if we did this:  https://core.trac.wordpress.org/ticket/39996 - and I suspect there are probably still a few more.